finality-aleph/src/sync/substrate/verification/cache.rs

```diff
@@ -38,7 +38,7 @@ impl Display for CacheError {
             UnknownAuthorities(session) => {
                 write!(
                     f,
-                    "authorities for session {:?} not present on chain even though they should be",
+                    "authorities for session {:?} not present on chain. Most likely state is already pruned",
```

Comment:
The older message was better. We expected the authorities to be present. Semi-guessing that they are pruned even though the running might not be turned on could be misleading.

Reply:
True, that said, chain part of the message is also confusing now. How about "authorities for session {:?} not known even though they should be"?